docs/guides/universal-receiver-delegate/create-custom-urd-1.md

```diff
@@ -238,7 +232,7 @@ contract LSP1URDForwarderMethod1 is
         bytes4 interfaceId
     ) public view virtual override returns (bool) {
         return
-            interfaceId == _INTERFACEID_LSP1 ||
+            interfaceId == _INTERFACEID_LSP1_DELEGATE ||
```

Comment:
same, I don't find it in the constants file

Reply:
This is coming with this PR: https://github.com/lukso-network/lsp-smart-contracts/pull/741